src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -45,7 +45,8 @@
   ],
   "dependencies": {
     "@microsoft/signalr": "link:../signalr",
-    "@msgpack/msgpack": "^2.7.0"
+    "@msgpack/msgpack": "^2.7.0",
+    "eventsource": "^2.0.2"
```

Comment:
Why is this being added? msgpack has no need for an eventsource dependency

Reply:
Weird, `yarn install` did everything for me

Reply:
Updated yarn locally & it seems to be doing the right thing now